js/interface.js

```diff
@@ -686,36 +698,41 @@ $('input[type="file"]').on('change', function() {
 $('.redirectToSettings, [data-change-settings]').on('click', function(event) {
   event.preventDefault();
 
-  Fliplet.Studio.emit('close-overlay', {
-    name: 'publish-google'
+  saveProgressOnClose().then(function () {
+    Fliplet.Studio.emit('close-overlay', {
+      name: 'publish-google'
+    });
+  
+    Fliplet.Studio.emit('overlay', {
+      name: 'app-settings',
+      options: {
+        size: 'large',
+        title: 'App Settings',
+        section: 'appSettingsGeneral',
+        appId: Fliplet.Env.get('appId')
+      }
+    });
   });
```

Comment:
There should be a `.catch()` clause like what you've done for iOS AAB UI.

Reply:
@tonytlwu 
Fixed